### PR TITLE
Make game board responsive

### PR DESCRIPTION
**Pull request by apomelitos**

### CODE REVIEW

File: src/components/VideoControl.tsx [hunk 1]
```diff
@@ -100,8 +100,15 @@ export const VideoControl: FC<VideoControlProps> = ({
     };
   }, [stream, video]);
 
+  const styles = {
+    border: isVideoEnabled ? '1px solid lightgray' : 'none',
+    maWidth: '100%',
+    textAlign: 'center',
+    padding: 10,
+  } as React.CSSProperties;
```
Comment (mrlika) [inline src/components/VideoControl.tsx]: Static CSS properties should go in CSS classes. In this case I think you can add them to `video-wrapper`

File: src/components/Tile.tsx [hunk 2]
```diff
@@ -8,18 +8,20 @@ type TileProps = {
...
 
-const positionToPixels = ([row, col]: number[]): TopLeftStyles => {
+const positionToPixels = ([row, col]: [number, number]): TopLeftStyles => {
+  const tileMargin = window.innerWidth <= 500 ? 5 : 10;
+
   return {
-    top: row * 100 + 10,
-    left: col * 100 + 10,
+    top: `calc(${row} * ((100% - 20px) / 4) + ${tileMargin}px)`,
+    left: `calc(${col} * ((100% - 20px) / 4) + ${tileMargin}px)`,
```
Comment (mrlika) [inline src/components/Tile.tsx]: Refactor to pure CSS

File: src/components/Tile.scss [hunk 3]
```diff
@@ -7,15 +7,18 @@ $tile-size: 80px;
...
 
 .tile {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
   position: absolute;
-  width: $tile-size;
-  height: $tile-size;
+  width: calc(((100% - 20px) / 4 - 20px));
```
Comment (mrlika) [inline src/components/Tile.scss]: `box-sizing: border-box` in many cases allows to handle margins without `calc`: https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

File: src/components/Game.tsx [hunk 4]
```diff
@@ -99,6 +95,18 @@ export const Game: FC = (): JSX.Element => {
...
+    setTiles((prev) => [...prev]);
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', onResizeWindowHandler);
+
+    return () => {
+      window.removeEventListener('resize', onResizeWindowHandler);
+    };
+  }, []);
```
Comment (mrlika) [inline src/components/Game.tsx]: Looks like a hack to force re-render and recalculation of sizes in JS on resize

**Review by mrlika — CHANGES_REQUESTED** [review]

**Review by TheForsakenSpirit — APPROVED** [review]

**Pull request merged**